notebooks/2019-11-12_Calendars_and_monthly_data.ipynb

```diff
@@ -0,0 +1,576 @@
+    "\n",
+    "    # We rename the time dimension and coordinate to time360 to make it clear it isn't \n",
+    "    # the original time coordinate.\n",
+    "    val = val.rename({'time':'time360'})\n",
+    "    time1 = time1.rename({'time':'time360'})\n",
+    "    val = val.assign_coords({'time360':time1})\n",
+    "    return val"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<xarray.DataArray 'time360' (time360: 300)>\n",
+      "array([cftime.Datetime360Day(2274, 12, 16, 0, 0, 0, 0, 6, 346),\n",
+      "       cftime.Datetime360Day(2275, 1, 16, 0, 0, 0, 0, 1, 16),\n",
+      "       cftime.Datetime360Day(2275, 2, 16, 0, 0, 0, 0, 3, 46), ...,\n",
+      "       cftime.Datetime360Day(2299, 9, 16, 0, 0, 0, 0, 5, 256),\n",
+      "       cftime.Datetime360Day(2299, 10, 16, 0, 0, 0, 0, 0, 286),\n",
+      "       cftime.Datetime360Day(2299, 11, 16, 0, 0, 0, 0, 2, 316)], dtype=object)\n",
+      "Coordinates:\n",
+      "  * time360   (time360) object 2274-12-16 00:00:00 ... 2299-11-16 00:00:00\n",
+      "    model_id  <U10 'HadGEM2-ES'\n",
+      "Attributes:\n",
+      "    bounds:         time_bnds\n",
+      "    axis:           T\n",
+      "    long_name:      time\n",
+      "    standard_name:  time\n",
+      "<xarray.DataArray 'time360' (time360: 2412)>\n",
+      "array([cftime.Datetime360Day(2100, 1, 16, 0, 0, 0, 0, 1, 16),\n",
+      "       cftime.Datetime360Day(2100, 2, 16, 0, 0, 0, 0, 3, 46),\n",
+      "       cftime.Datetime360Day(2100, 3, 16, 0, 0, 0, 0, 5, 76), ...,\n",
+      "       cftime.Datetime360Day(2300, 10, 16, 0, 0, 0, 0, 3, 286),\n",
+      "       cftime.Datetime360Day(2300, 11, 16, 0, 0, 0, 0, 5, 316),\n",
+      "       cftime.Datetime360Day(2300, 12, 16, 0, 0, 0, 0, 0, 346)], dtype=object)\n",
+      "Coordinates:\n",
+      "  * time360   (time360) object 2100-01-16 00:00:00 ... 2300-12-16 00:00:00\n",
+      "    model_id  <U10 'bcc-csm1-1'\n",
+      "Attributes:\n",
+      "    bounds:         time_bnds\n",
+      "    axis:           T\n",
+      "    long_name:      time\n",
+      "    standard_name:  time\n"
+     ]
+    }
+   ],
+   "source": [
+    "ll = [to_360day_monthly(da) for da in var_list]\n",
+    "print(ll[0].time360)\n",
+    "print(ll[1].time360)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now have the same calendar and all dates refer to the middle of the month. We can concatenate the arrays together with `xarray.concat()` and xarray will align the arrays correctly. Make sure the dates that are common to both models are the same, otherwise Xarray might keep both sets. This would result in additional times in both models filled with NaN. Additionally, the data from both models would never be considered having the same time and as such you wouldn't be able to do a mean."
```

Comment:
Additionally, the data from both models would never be considered **as** having the same time